browser/src/control/Control.JSDialog.js

```diff
@@ -145,6 +145,7 @@ L.Control.JSDialog = L.Control.extend({
 				console.warn('closePopover: no builder');
 		}
 		else {
+			JSDialog.CloseDropdown(id, true);
```

Comment:
not sure if it is needed, how does the call stack look like for this bug?

Reply:
(anonymous) (Widget.Combobox.js:327)
fire (Events.js:146)
JSDialog.CloseDropdown (Util.Dropdown.js:161)
closePopover (Control.JSDialog.js:148)
close (Control.JSDialog.js:86)
(anonymous) (Control.JSDialog.js:227)